.circleci/config.yml

```diff
@@ -0,0 +1,40 @@
+version: 2
+env:
+  global:
+  - LC_CTYPE=en_US.UTF-8
+  - LANG=en_US.UTF-8
+jobs:
+  test-iOS:
+    macos:
+      xcode: "9.3.0"
```

Comment:
Can we run it with Xcode 10?

Reply:
Sure. I did not want to change anything in the config right now. I can do it in another PR.